internal/ansible/flags/flag.go

```diff
@@ -162,6 +163,11 @@ func (f *Flags) AddTo(flagSet *pflag.FlagSet) {
 		"The amount of time that will be spent waiting"+
 			" for runners to gracefully exit.",
 	)
+	flagSet.StringVar(&f.AnsibleLogEvents,
+		"ansible-log-events",
+		"",
+		"Ansible log events. The log level for console logging.",
```

Comment:
You should call out here what the options are, and set the default to tasks probably.